packages/teleport/src/Login/ErrorPage/ErrorPage.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
```

Comment:
You can call this page as IdemeumErrorPage/ErrorPage.tsx 
Do not delete the old file, when we rebase with latest webapps we dont have to worry about it.

Reply:
Folder Renamed and Code committed